### ISSUE TITLE
BUG | Changed password (safety practices) Windows (8) application now gives error; [within app.asar]

### ISSUE THREAD

**Issue by ninetyfivenorth** [mymonero/mymonero-app-js#386]
Changed password (safety practices) Windows (8) application now gives error; [within app.asar]

![mymonero-bug](https://user-images.githubusercontent.com/31463232/120371950-939ffd80-c2e4-11eb-80eb-ebf57abeca16.png)
![mymonero-bug_2](https://user-images.githubusercontent.com/31463232/120371957-969aee00-c2e4-11eb-89db-d97cffbe6fbf.png)
![mymonero-bug_3](https://user-images.githubusercontent.com/31463232/120371970-98fd4800-c2e4-11eb-9c20-ed4c8609fd60.png)

I assume with the password change, something happened within the wallet's details that its now confused by the change and still looking for the old password but sees that the new one I enter is right but not right like it _once_ was lol

Let me know how I can fix, 
Or if I am completely fucked and lost all of my money.

Thanks guys @eugeneotto @charleyhine @paulshapiro @devinpearson @fluffypony

**Comment by ninetyfivenorth** [comment]
I am contemplating re-installing the application and then just re-entering my mnemonic seed and calling it a day.

As the password is only directly correlated to the use of the application upon the OS itself, correct?

I'm also thinking, because the application is still presenting me with the option of changing the pin/password upon re-opening the program each time (at this time)... that if I changed the password back to the original one, the .js might like seeing that once again and be happy with that && the wallet data within.

Thoughts?

**Comment by devinpearson** [comment]
Hi Shane, 

It does look like it's corrupted the save file. 

Hardening the save process is on our roadmap. It currently writes to the files at least every 30 seconds which is not ideal. 

Reinstalling the app won't clear out the files. You can however click the settings menu and scroll o the bottom there is a delete all button. If that's not possible you can find the file location by going to the top menu. Click help / Users data folder. Delete the files found there and restart the app. You can now open your wallet with your mnemonic seed again

**Comment by ninetyfivenorth** [comment]
@devinpearson Thanks for the quick response. Yea I do have the option to use the delete everything button... I was simply hesitant about using it as "everything" in my mind could mean so much as deleting the wallet itself entirely. I prefer not to use the web interface due to my own choice of safe practices online (especially with crypto) but should I try to connect to my wallet using my seed online just for shits and then do the wipe everything of the app and then go through the process of re-adding said same wallet using the seed and add a new (obv) password for the client itself

Thoughts?

**Comment by ninetyfivenorth** [comment]
P.S. I did end up looking up the **app.asar** file and how to go about entering it. Such as, I have added the plugin to 7zip and extracted the .asar file to do a little snooping around myself of the errors listed being posted using the UI.  

My question, is it a **security risk exposing my system to the contents of the .asar?** I quickly deleted the dir in which I used to extract the app.asar file into and have a copy of the contents saved within a password protected .zip archive. Is it vulnerable for I having entered the .asar file and now having its contents within a password protected archive?

Thoughts?
